subsys/bluetooth/host/hci_core.h

```diff
@@ -159,6 +159,11 @@ enum {
 	/** Periodic advertising is attempting sync sync */
 	BT_PER_ADV_SYNC_RECV_DISABLED,
 
+	/* Constant Tone Extension for Periodic Advertising has been enabled
```

Comment:
```suggestion
	/** Constant Tone Extension for Periodic Advertising has been enabled
```
Not really needed, but would make it more consistent

Reply:
Corrected